api/src/main/java/org/pmiops/workbench/featuredworkspace/FeaturedWorkspaceServiceImpl.java

```diff
@@ -29,4 +45,37 @@ public Optional<FeaturedWorkspaceCategory> getFeaturedCategory(DbWorkspace dbWor
                 featuredWorkspaceMapper.toFeaturedWorkspaceCategory(
                     dbFeaturedWorkspace.getCategory()));
   }
+
+  @Override
+  public void backFillFeaturedWorkspaces() {
+    FeaturedWorkspacesConfig fwConfig = featuredWorkspacesConfigProvider.get();
+    // Get featured workspaces from the config
+    FeaturedWorkspacesConfigResponse configResponse =
+        new FeaturedWorkspacesConfigResponse().featuredWorkspacesList(fwConfig.featuredWorkspaces);
+    configResponse
+        .getFeaturedWorkspacesList()
+        .forEach(
+            fw -> {
+              String workspaceNamespace = fw.getNamespace();
+              String firecloudName = fw.getId();
+              try {
+                // Get Dbworkspace from workspaceNamesapce and firecloudname
+                DbWorkspace dbWorkspace =
+                    workspaceDao.getRequired(workspaceNamespace, firecloudName);
+
+                if (dbWorkspace.getPublished()) {
```

Comment:
I don't think this is a reliable flag, so I'd prefer to skip this check.

Reply:
I wonder if there is a case where a workspace was 'unpublished' but the entry was left in configuration

Reply:
It's possible - but what would we want to do in that case?  Which is correct?

I'd argue that it's more likely that the config is correct, because it goes through a formal release and change process.  Publish/unpublish is just clicking a button, so I think that is the more error-prone value.